src/workers/jobs.js

```diff
@@ -60,6 +64,62 @@ export async function uploadContacts(job) {
   }
 }
 
+
+export async function loadContactsFromDataWarehouse(job) {
+  let sql_query = job.payload
+  if (!sql_query.startsWith('SELECT') || sql_query.indexOf(';') >= 0) {
+    log.error('Malformed SQL statement.  Must begin with SELECT and not have any semicolons: ', sql_query)
+    return
+  }
+  if (!datawarehouse) {
+    log.error('No data warehouse connection, so cannot load contacts', job)
+    return
+  }
+
+  knexResult = await datawarehouse.raw(sql_query)
+  let fields = {}
+  let customFields = {}
+  const contactFields = {
+    first_name: 1,
```

Comment:
the value doesn't matter.  I just want to test whether it's in there.  I don't think es6 has sets, but i bet if i google it, they will.